Why is this code here? Without it, strange things happend....
	--jsaw

--- bash-3.0/variables.c	2005-01-03 23:28:24.496779416 +0100
+++ bash-3.0-t2/variables.c	2005-01-04 05:16:11.586499592 +0100
@@ -1260,7 +1260,7 @@
   if (rl_completer_word_break_characters == 0 && bash_readline_initialized == 0)
     enable_hostname_completion (perform_hostname_completion);
 
-#if 0
+#if 1
   FREE (value_cell (var));
   p = savestring (rl_completer_word_break_characters);
   
@@ -1386,7 +1386,7 @@
 #else
   INIT_DYNAMIC_VAR ("FUNCNAME", (char *)NULL, get_funcname, null_assign);
 #endif
-  VSETATTR (v, att_invisible|att_noassign);
+  VSETATTR (v, att_invisible|att_noassign|att_nounset);
   return v;
 }
 
